scripts/destroy.sh

```diff
@@ -13,6 +15,12 @@ kubectl delete --ignore-not-found --wait=true -R -f platform/
 # Remove HelmOperator 
 helm delete --purge helm-operator
 
+# Install block storage incase of IBM
+if [[ $CLOUD_PROVIDER == "ibm" ]];
+then
+  helm delete --purge ibmc-block-storage
+fi
+
```

Comment:
This is not installing, rather removing. Also, they have `ibmc-block-storage` installed by default no ?

Reply:
If you are upgrading then this shouldn't be removed i think